src/python/WMCore/Storage/Backends/GFAL2Impl.py

```diff
@@ -113,27 +114,78 @@ def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=No
                 copyCommandDict['checksum'] = "-K adler32"
 
         copyCommandDict['options'] = ' '.join(unknown)
-
         copyCommandDict['source'] = self.createFinalPFN(sourcePFN)
         copyCommandDict['destination'] = self.createFinalPFN(targetPFN)
 
         copyCommand = self.copyCommand % copyCommandDict
         result += copyCommand
 
+        logging.debug("Actual command which failed: %s", copyCommand)
```

Comment:
We don't know if the command failed (retries > 0) or not (retry=0), so it is a misleading message.